tests/fixtures/ptx/gnu_ext_disabled_auto_reg_head_truncate.expected

```diff
@@ -0,0 +1,114 @@
+.xx "too short a date:" "" "And summer's lease hath all" "" "sonnet:4"
```

Comment:
please generate some contents on the fly, don't add test files
it isn't super interesting to have this in the repo